frontend/app/src/components/settings/data-security/backups/ManageCustomAssets.vue

```diff
@@ -48,54 +48,60 @@ const { t } = useI18n();
   <Card>
     <template #title>{{ t('manage_user_assets.title') }}</template>
 
-    <VAlert type="info" outlined dense>
-      {{ t('manage_user_assets.warning') }}
-    </VAlert>
+    <div class="flex flex-col gap-4">
+      <RuiAlert type="info" dense>
+        {{ t('manage_user_assets.warning') }}
+      </RuiAlert>
 
-    <VSheet outlined class="pa-4" rounded>
-      <div class="text-h6">{{ t('manage_user_assets.export.title') }}</div>
-      <div class="text-subtitle-1">
-        {{ t('manage_user_assets.export.subtitle') }}
-      </div>
-      <VAlert v-if="exportError" type="error" outlined dense>
-        {{ exportError }}
-      </VAlert>
-      <div class="flex flex-row items-center mt-4">
-        <VBtn color="primary" :loading="downloading" @click="exportZip()">
-          {{ t('manage_user_assets.export.button') }}
-        </VBtn>
-        <VIcon v-if="downloaded" class="ms-4" color="success">
-          mdi-check-circle
-        </VIcon>
-        <span v-if="downloaded" class="ms-2">
-          {{ t('manage_user_assets.export.success') }}
-        </span>
-      </div>
-    </VSheet>
+      <RuiCard outlined>
+        <div class="text-h6">{{ t('manage_user_assets.export.title') }}</div>
```

Comment:
@lukicenturi let's use the header and subheader from the card, and we need to adjust the RuiCard to do text-h6 instead of h5